presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt

```diff
@@ -1,36 +1,68 @@
 #$ TITLE: SupaflyFPV Freestyle 5 Inch
-#$ FIRMWARE_VERSION: 4.3
+#$ FIRMWARE_VERSION: 4.3, 4.4
 #$ CATEGORY: TUNE
 #$ STATUS: OFFICIAL
 #$ KEYWORDS: Supafly, SupaflyFPV, 4S, 6S, Freestyle, 5 inch, 5"
 #$ AUTHOR: SupaflyFPV
-#$ DESCRIPTION: SupaflyFPV Community Freestyle Preset for 5 Inch Builds
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: [<img src="https://supaflyfpv.com/wp-content/uploads/2022/06/Logo-Cropped192.svg" width="220px" style="margin-left: auto; margin-right: auto; display: block;"/>](https://supaflyfpv.com)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
 #$ DESCRIPTION:
-#$ DESCRIPTION: Select the option for your build with or without HD camera (eg GoPro) as the baseline.
+#$ DESCRIPTION: ### SupaflyFPV 5 Inch Freestyle Preset
 #$ DESCRIPTION:
-#$ DESCRIPTION: Half to one click up or down Master Slider to tweak the tune - in OSD or Configurator.
+#$ DESCRIPTION: <br>
 #$ DESCRIPTION:
-#$ DESCRIPTION: For a Master Slider setting tailored to your Lipo and Motor Kv go here: https://tinyurl.com/SupaflyFPV
+#$ DESCRIPTION: #### New Higher Performance! Pick your options from the drop-down:
 #$ DESCRIPTION:
-#$ DESCRIPTION: RPM Filtering for best performance (default) - or de-select for Non RPM Setup (check esc compatibility).
+#$ DESCRIPTION: <br>
 #$ DESCRIPTION:
-#$ DESCRIPTION: If using F7/H7 Flight Controller run 8k Pidloop and turn off Gyro Low Pass Filter 2 for max performance.
+#$ DESCRIPTION: * RPM Filtering recommended on Blheli32 esc or de-select if ESC is not compatible
+#$ DESCRIPTION: * Apply 'Rough Motors' option if your motors or frame are old/beat up
+#$ DESCRIPTION: * Pick starter PIDs option - 4s works for 2400-2700kv motors or 6s for 1750 - 1950kv
+#$ DESCRIPTION: * For further optimisation to your specific motor kv [use this custom PID calculator](https://supaflyfpv.com/betaflight-setup/)
+#$ DESCRIPTION: 
+#$ DESCRIPTION: <br>
 #$ DESCRIPTION:
 #$ DESCRIPTION: Enjoy!
-#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/124
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: Support, feedback and chat:
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION:*[FACEBOOK SUPPORT/FEEDBACK](https://www.facebook.com/groups/supaflyfpv/)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION:*[DISCORD SUPPORT/FEEDBACK](https://discord.gg/Gs5nTx4rE2)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: 
 #$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
 #$ INCLUDE: presets/4.3/tune/defaults.txt
 #$ INCLUDE: presets/4.3/filters/defaults.txt
 
+# -- Logging --
+
+set blackbox_sample_rate = 1/4
+set debug_mode = GYRO_SCALED
```

Comment:
plz remove

Reply:
this is useful for troubleshooting down the line should it be necessary - why the need to remove?

Reply:
A person might have his blackbox already setup, or he simply does not want it.

The general rule, we can't set parameters that are not part of the tune/filters outside of the unchecked options.

Reply:
Ok, the categorisation issue...how about some debug presets? If so what category should they go in...

Reply:
we already have some:
https://github.com/betaflight/firmware-presets/blob/master/presets/4.3/other/blackbox_disable.txt

Reply:
ok but I think I'll make a troubleshoot preset or something...that just disables parameters for more space...

Reply:
It should be set as unchecked an option as it fall out side of what a tune is